### PR TITLE
fix #1141

### LINKED ISSUE #1141: Cannot animate Isofill
When you try to animate an isofill, you receive this error:

```
/Users/fries2/uvcdat/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/VTKPlots.pyc in update_input(self, vtkobjects, array1, array2, update)
   1797                   else:
   1798                       mapper = vtk.vtkPolyDataMapper()
-> 1799                       mapper.SetInputConnection(ports[i].GetOutputPort())
   1800                       lut,rg = vtkobjects["vtk_backend_luts"][i]
   1801                       if lut is not None:

IndexError: list index out of range
```

Test case:

```python
import vcs, cdms2 as cdms, os, sys

# Open data file:
filepath = os.path.join(vcs.prefix, 'sample_data/clt.nc')
cdmsfile = cdms.open( filepath )
x = vcs.init()
x.isofill(cdmsfile["clt"])
x.animate.create()
x.animate.run()
x.animate.stop()
```

### PR DESCRIPTION
**Pull request by doutriaux1**
these lines seems to be left over from tests for hatch/pattern and create the animation confusion

@aashish24 @chaosphere2112 please review and merge

### CODE REVIEW

**Comment by doutriaux1** [comment]
@chaosphere2112 do you feel like merging this one?

File: Packages/vcs/Lib/VTKPlots.py [hunk 1]
```diff
@@ -1008,8 +1008,8 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
           #png.SetFileName("/git/uvcdat/Packages/vcs/Share/uvcdat_texture.png")
           #T=vtk.vtkTexture()
           #T.SetInputConnection(png.GetOutputPort())
-          if isinstance(gm,isofill.Gfi):
-              mappers.append([mapper,])
+          #if isinstance(gm,isofill.Gfi):
```
Comment (aashish24) [inline Packages/vcs/Lib/VTKPlots.py]: can we take it out. Why leave this code over?

Comment (doutriaux1) [inline Packages/vcs/Lib/VTKPlots.py]: becauseit's code i left here for you guys when you will be doing the hatch/patterns, feel free to take it out.

Comment (doutriaux1) [inline Packages/vcs/Lib/VTKPlots.py]: @aashish24 did you run ctest? I was talking about the commented out lines above. I'm pretty sure these lines are needed. Othwerwise your mappers will be empty and nothing would be drawn? no?

**Pull request merged**